src/mongocxx/test/database.cpp

```diff
@@ -205,4 +219,13 @@ TEST_CASE("Database integration tests", "[database]") {
 
         clear_collection(database, collection_name);
     }
+
+    SECTION("A database may be dropped") {
+        clear_collection(database, collection_name);
+
+        database.create_collection(collection_name);
+        REQUIRE(database.has_collection(collection_name));
+        database.drop();
+        REQUIRE(!database.has_collection(collection_name));
```

Comment:
I wasn't sure of the best way to test whether a database had been dropped. I also considered iterating through list_databases. Any advice on this?